SU-SWS/Stanford-Drupal-Profile
# DEVOPS-144: Update Drupal core to 7.58

# READY FOR REVIEW

# Summary
- https://www.drupal.org/sa-core-2018-002

# Needed By (Today)

# Urgency
- I'm going to say 10, so new sites get the 7.58 version of Drupal.

# Steps to Test

Travis results:
4 `Waiting for AJAX to finish` failures
1 missing file failure: https://travis-ci.org/SU-SWS/Stanford-Drupal-Profile/builds/358893232#L3210.  This failure showed up before the 7.58 update, see: https://travis-ci.org/SU-SWS/Stanford-Drupal-Profile/jobs/358907217#L3211.

And seems to warrant further exploration at a later point, as the file appears to have been copied and should have been available to the test: https://travis-ci.org/SU-SWS/Stanford-Drupal-Profile/builds/358893232#L1100 and https://travis-ci.org/SU-SWS/Stanford-Drupal-Profile/jobs/358907217#L1101.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-144
